Cargo.toml

```diff
@@ -12,9 +12,15 @@ members = [
 inherits = "release"
 lto = true
 
+[profile.release]
+panic = "abort"
```

Comment:
You should add this line to `release-lto` and to `integration-test` profiles as well. The only profile, that should preserve unwind behavior is `test`

Reply:
`release-lto` inherits `release`, so no reason to duplicate it here. I'll add one to `integration-test`

Reply:
Done